src/renderers/PartialViewRenderers/region/facilitiesInfo.js

```diff
@@ -7,14 +7,27 @@ $(function () {
     addFacility_handler();
     //handle delete facility
     deleteFacility_handler();
+    //handle change in filters in facility
```

Comment:
Why is there a comment? If the code requires comments that means it's not clear enough and it's the code that needs to be changed

Reply:
I just follow the predecessor usage of comments there

Reply:
I will remove it

Reply:
> I just follow the predecessor usage of comments there

Only do this if the code is very complex. If it is changeable, then change it. `Leave code better than when you found it`